src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -263,8 +266,32 @@ angular
  * </md-autocomplete>
  * </hljs>
  *
+ * ### Specifying Repeat Mode
+ * You can use md-mode to specify whether to use standard or virtual lists for
+ * rendering autocomplete suggestions.
+ * The md-mode accepts two values:
+ * - `virtual` (default) Uses md-virtual-repeat to render list items. If the
+ * - `standard` uses ng-repeat to render list items
+ *
+ * Note that using 'standard' mode will require you to address any list
+ * performance issues (e.g. pagination) separately within your application.
+ *
+ * <hljs lang="html">
+ *   <md-autocomplete ... md-mode="standard">
```

Comment:
Let's make this template a little more complete based on the basic example above.